docs/ref/settings.txt

```diff
@@ -1836,18 +1836,23 @@ Default::
 
     {}  # empty dictionary
 
-A dictionary specifying the package where migration modules can be found on a per-app basis. The default value
-of this setting is an empty dictionary, but the default package name for migration modules is ``migrations``.
+A dictionary specifying the package where migration modules can be found on a
+per-app basis. The default value of this setting is an empty dictionary, but
+the default package name for migration modules is ``migrations``.
 
 Example::
 
     {'blog': 'blog.db_migrations'}
 
-In this case, migrations pertaining to the ``blog`` app will be contained in the ``blog.db_migrations`` package.
+In this case, migrations pertaining to the ``blog`` app will be contained in
+the ``blog.db_migrations`` package.
 
 If you provide the ``app_label`` argument, :djadmin:`makemigrations` will
 automatically create the package if it doesn't already exist.
 
+When you supply ``None`` as a value for an app Django will consider the app as
```

Comment:
My opinion is that it's better to explain why it exists and discourage its use except for certain cases instead of leaving the reader wondering about it.